src/data/faq_de.json

```diff
@@ -718,17 +718,15 @@
-                            "Samsung bietet für ihre Geräte zusätzlich einen sogenannten \"Ultra-Energiesparmodus\" an. Dieser verhindert oft eine erfolgreiche Hintergrundaktualisierung, selbst wenn 'Priorisierte Hintergrundaktivität' aktiviert ist. <b>Für eine ordnungsgemäße Funktion der Corona-Warn-App sollte der Ultra-Energiesparmodus daher ausgeschaltet sein.</b> Beachten Sie dazu die <a href='#max_power_save' target='_blank' rel='noopener'>Hinweise zum Ultra-Energiesparmodus auf Samsung-Geräten</a>.",
-                            "Sie können in den Energiespar-Einstellungen Ihres Smartphones sicherstellen, dass für die Corona-Warn-App eine Ausnahme vom Energiesparen festgelegt ist:",
+                            "<ol><li>Wählen Sie in der Corona-Warn-App (CWA) auf der Startseite das Dreipunktmenü oben rechts, dann 'Einstellungen'</li><li>Wählen Sie 'Priorisierte Hintergrundaktivität' (Sollte Priorisierte Hintergrundaktivität schon 'An' sein, brauchen Sie hier nichts weiter tun.)</li><li>Wählen Sie 'Smartphone-Einstellungen öffnen'</li><li>Sie sehen jetzt das Betriebssystem-Menü 'Akkuverbrauch optimieren'/'Akku-Optimierung'/'Akkuoptimierung'. </li><li>Wählen Sie in der Dropdown-Liste 'Alle' oder 'Alle Apps'</li><li>Scrollen zu runter zu 'Corona-Warn'</li><li>Ziehen Sie den Schieberegler nach links (d.h. 'Akkuverbrauch optimieren aus') oder tippen Sie auf den Eintrag und wählen Sie dann 'Nicht optimieren' aus. (Dieser Vorgang ist unterschiedlich je nach Gerätemodell)</li><li>Überprüfen Sie die Änderung in der CWA: Gehen Sie in der CWA wieder über das Dreipunktmenü oben rechts in die Einstellungen.</li><li>Priorisierte Hintergrundaktivität sollte nun 'An' sein</li></ol>",
+                            "Samsung bietet für Ihre Geräte zusätzlich einen sogenannten \"Ultra-Energiesparmodus\" an. Dieser verhindert oft eine erfolgreiche Hintergrundaktualisierung, selbst wenn die 'priorisierte Hintergrundaktivität' aktiviert ist. <b>Für eine ordnungsgemäße Funktion der Corona-Warn-App sollte der Ultra-Energiesparmodus daher ausgeschaltet sein.</b> Beachten Sie dazu die <a href='#max_power_save' target='_blank' rel='noopener'>Hinweise zum Ultra-Energiesparmodus auf Samsung-Geräten</a>.",

```

Comment:
Auf manchen Samsung Geräte heißt es nicht "Ultra-Energiesparmodus" sondern "Maximaler Energiesparmodus", z. B. Samsung Galaxy A5 (2017) mit Android 8.

In English it is called "Maximum power saving mode".

"Samsung bietet für Ihre Geräte zusätzlich einen sogenannten \"Ultra-Energiesparmodus\" an." => 
"Samsung bietet für Ihre Geräte zusätzlich einen sogenannten \"Ultra-Energiesparmodus\" an, bei manchen Geräten als \"Maximaler Energiesparmodus\" bezeichnet."

Reply:
ok, great. Thank you!